testutil/context.go

```diff
@@ -24,3 +24,18 @@ func DefaultContext(key storetypes.StoreKey, tkey storetypes.StoreKey) sdk.Conte
 
 	return ctx
 }
+
+func DefaultContextWithDB(key storetypes.StoreKey, tkey storetypes.StoreKey) (sdk.Context, *dbm.MemDB, store.CommitMultiStore) {
```

Comment:
I would recommend we return a type with these three values instead. Perhaps a `TestContext struct`.